cmd/sshkey_list.go

```diff
@@ -73,6 +70,22 @@ func listSSHKey(t *table.Table, filters []string) error {
 	return nil
 }
 
+func getSSHKeys(cs *egoscale.Client) ([]egoscale.SSHKeyPair, error) {
+	sshKeys, err := cs.ListWithContext(gContext, &egoscale.SSHKeyPair{})
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]egoscale.SSHKeyPair, 0, len(sshKeys))
+
+	for _, key := range sshKeys {
+		k := key.(*egoscale.SSHKeyPair)
+		res = append(res, *k)
```

Comment:
we can go with: `res[i] = *k`

Reply:
It's the same because I have put that `res := make([]egoscale.SSHKeyPair, 0, len(sshKeys))`
but okay if you want